.github/workflows/documentation.yml

```
@@ -0,0 +1,22 @@
+name: Documentation
+on:
+  push:
+    branches: [ $default-branch ]
+
+jobs:
+  deploy-documentation:
+    runs-on: ubuntu-16.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Dokka
+        run: ./gradlew dokkaHtmlMultimodule
```

Being that lowercase `m` is deprecated, let's move to capital (assume it works the same).

```suggestion
        run: ./gradlew dokkaHtmlMultiModule
```